docs/topics/circuit-breaker-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -0,0 +1,104 @@
+= Interact with the Unmodified {SpringBoot} Booster
+
+Once you have the {SpringBoot} booster deployed, you have the following services running:
+
+//{app-name} = springboot-cb
+
+`{app-name}-name`::
+Exposes the following endpoints:
+
+* the `/api/name` endpoint, which returns a name when this service is working, and an error when this service is set up to demonstrate failure.
+
+* the `/api/state` endpoint, which controls the behavior of the `/api/name` endpoint and determines whether the service works correctly or demonstrates failure.
+
+`{app-name}-greeting`::
+Exposes the following endpoints:
+
+* the `/api/greeting` endpoint that you can call to get a personalized greeting response.
++
+Calling the `/api/greeting` endpoint also calls the `/api/name` endpoint of the `{app-name}-name` service.
+This call happens behind the scenes and is protected by the Circuit Breaker.
```

Comment:
I agree my original wording wasn't terribly awesome here. What I wanted to convey by the "behind the scenes" words is: as part of processing of the `/api/greeting` endpoint, the `{app-name}-greeting` service calls the `/api/name` endpoint of the `{app-name}-name` service. This call is guarded by the circuit breaker.

To me, the sentence "This call happens behind the scenes and is protected by the Circuit Breaker" places an equal emphasis on "happens behind the scenes" and "is protected by the circuit breaker", which isn't really appropriate. The really important thing here is that the call is guarded by the circuit breaker. The fact that `{app-name}-greeting` calls into `{app-name}-name` is just a description of how the services interact.

IDK, can we improve the wording somehow? Maybe I'm just nitpicking here :-)